data/json/mutations/mutations.json

```diff
@@ -4495,7 +4495,7 @@
     "points": -4,
     "visibility": 8,
     "mixed_effect": true,
-    "description": "You're only half as tall as you used to be!  The weight of things you once found easy to carry is now unbearable, clothes are now twice as encumbering for you unless you refit them (since you're half their size), and your hit points are heavily reduced.  However, your movement is silent, and your dodge skill is a little higher.",
+    "description": "You are only half as tall as you used to be!  The weight of things you once found easy to carry is now unbearable, clothes are now twice as encumbering for you unless you refit them (since you're half their size), and your hit points are heavily reduced.  However, your movement is silent, and your dodge skill is a little higher.",
```

Comment:
That's not an error?

Reply:
AFAIK, technically, `You're` is correct too, it was highlighted for a different reason (false positive) and I've decided to tune it. Should I revert it?

Reply:
Yes, please. The change is minor, but it'll break the existing translation with no profit.